core-java-modules/core-java-arrays-sorting/src/test/java/com/baeldung/sorted/operations/InsertElementInSortedArrayTest.java

```diff
@@ -0,0 +1,18 @@
+package com.baeldung.sorted.operations;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+public class InsertElementInSortedArrayTest {
```

Comment:
Are there edge cases we should be testing? It seems insufficient to provide enough test coverage with a single unit test